src/Resources/views/edit_matrix.html.twig

```diff
@@ -24,6 +24,9 @@
                     {% for elem in record.children %}
                         <td>
                             {{ form_widget(elem) }}
+                            {% if 'array_text' in elem.vars.block_prefixes %}
```

Comment:
I just wonder, do we need this `if`? As default hint is empty, so probably it will not have any impact. And users will be able to add own hints to the fields.

Reply:
you're right, could be good to allow help for all type

Done